### PR TITLE
[openload] Fix extraction.

### PR DESCRIPTION
**Pull request by kasper93**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [x] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

The aadecode part is not mine, but I just restore the version which was removed here c1decda58c812b3d0a3d4dfa998e7d8bd8f99203

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

### CODE REVIEW

File: youtube_dl/extractor/openload.py [hunk 1]
```diff
@@ -56,6 +62,60 @@ class OpenloadIE(InfoExtractor):
...
+    @staticmethod
+    def openload_level2_debase(m):
+        radix, num = int(m.group(1)) + 27, int(m.group(2))
+        return '"' + encode_base_n(num, radix) + '"'
+
+    @classmethod
+    def openload_level2(cls, txt):
+        # The function name is ǃ \u01c3
+        # Using escaped unicode literals does not work in Python 3.2
+        return re.sub(r'ǃ\((\d+),(\d+)\)', cls.openload_level2_debase, txt, re.UNICODE).replace('"+"', '')
```
Comment (yan12125) [inline youtube_dl/extractor/openload.py]: ```ǃ``` function is no longer in obfuscated codes. Could you drop those unused program segments?

**Comment by yan12125** [comment]
Thanks! Sorry for delay

**Comment by Neney22** [comment]
Hey Did you find a fix to Openload? it seems like the latest version of Youtube-dl still returns 

> {"status":403,"msg":"wrong IP"}``
when trying to extract from the openload website

**Comment by yan12125** [comment]
Still working with the latest version:
```
$ youtube-dl -v https://openload.co/f/kUEfGclsU9o
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v', 'https://openload.co/f/kUEfGclsU9o']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.11.22
[debug] Git HEAD: 560c8c6
[debug] Python version 3.5.2 - Linux-4.8.10-1-ARCH-x86_64-with-arch
[debug] exe versions: ffmpeg 3.2, ffprobe 3.2
[debug] Proxy map: {}
[Openload] kUEfGclsU9o: Downloading webpage
[debug] Invoking downloader on 'https://openload.co/stream/kUEfGclsU9o~1480317710~140.112.0.0~i38quB0d?mime=true'
[download] Destination: skyrim_no-audio_1080.mp4-kUEfGclsU9o.mp4
[download] 100% of 9.80MiB in 00:11
```
Most likely you're not using the latest version.

**Comment by Neney22** [comment]
Yes I am using the latest version, youtube-dl does extracts the link correctly but if you try streaming the given link to download or stream the video as given in your debug: 

> 'https://openload.co/stream/kUEfGclsU9o~1480317710~140.112.0.0~i38quB0d?mime=true'

the error 

> '{"status":403,"msg":"wrong IP"}'

appears. Did you tried streaming the video?

**Comment by Hrxn** [comment]
http://jaimemf.github.io/youtube-dl-web/

Doesn't necessarily use the latest version of youtube-dl...

**Comment by Neney22** [comment]
I know that, but I am using the latest version on my site and also on an app uploaded to heroku, and like I said, it extracts the link but it returns that error on a browser. This issue is only with Open load every other link works fine..

**Comment by Hrxn** [comment]
Then I guess ('403') Heroku et al. IP ranges are blocked by openload..

**Comment by Neney22** [comment]
Take a look at their API https://openload.co/api I think it returns a download ticket based on the IP so Im guessing maybe youtube-dl is not generating the download link based on my IP or something haha

**Comment by yan12125** [comment]
You got it. Openload prohibits accessing across different IPs

**Pull request merged**